docs/source/cli_help/github.rst

```diff
@@ -64,9 +64,9 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
 
   Synchronizes with the remote repository:
 
-    * deletes untracked managed branches with no downstream branch,
```

Comment:
Pls also update the docs for `clean` to make the ordering of operations clear there